extra/1-currency-conversion.js

```diff
@@ -15,7 +17,10 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(pounds) {
+  let rounded = pounds * 5.7 * .99;
+  return Number(rounded.toFixed(2));
+}
 
```

Comment:
Hey Alex, I really like your solution for the convertToBRL function. I was stuck on this problem for some time, but I like how your solution is very similar to how the problem is written.